packages/status-js/src/client/client.ts

```diff
@@ -28,7 +28,7 @@ export interface ClientOptions {
    * Public key of a community to join.
    */
   publicKey: string
-  environment?: 'production' | 'test'
+  environment?: 'test' // 'production' | 'test'
```

Comment:
I don't fully understand the reason for changing to test so i guess it's necessary.

Reply:
We don't have a dedicated production yet – it's `shards.test` from https://fleets.status.im that's also used as production.